lib/core/utils/clone.js

```diff
@@ -8,6 +8,9 @@ function clone(obj) {
   var index,
     length,
     out = obj;
+  if (obj instanceof window.Node) {
```

Comment:
It might seem obvious, but a quick comment here might be helpful, as this code path does not "clone" anything.

```suggestion
  // DOM nodes cannot be cloned.
  if (obj instanceof window.Node) {
```

Reply:
In node `window.Node` doesn't exist so that's why the build is failing.

```suggestion
  if (window?.Node && obj instanceof window.Node) {
```